packages/nimble-components/src/theme-provider/design-token-names.ts

```diff
@@ -54,6 +54,7 @@ export const tokenNames: { readonly [key in TokenName]: string } = {
     dialogLargeWidth: 'dialog-large-width',
     dialogLargeHeight: 'dialog-large-height',
     dialogLargeMaxHeight: 'dialog-large-max-height',
+    treeViewLeftMargin: 'tree-view-left-margin',
```

Comment:
Build error is because "margin" isn't a valid "unit" which seems right.
I think I'd go for treeViewPaddingLeftSize (element tree-view, part padding) in a more pure world
We have legacy Padding as a "unit" even though it's not. If we pile on that tech debt it could be treeeViewLeftPadding

This is more accurately a padding (inside the component from the edge) vs margin (space outside the component definition of it's size).

Reply:
Ah, I was quite confused about that one - 👍